queue/abstract/driver/fifottl.lua

```diff
@@ -144,6 +144,7 @@ local function fifottl_fiber_iteration(self, processed)
     if estimated > 0 or processed > 1000 then
         -- free refcounter
         estimated = estimated > 0 and estimated or 0
+        processed = 0
```

Comment:
Hmm, I'm not sure this is a good place to reset the counter, because the function returns this value. Wouldn't it be better to reset the counter from the outside?

Reply:
I don't think so:

Now only one function uses the `processed` variable. In fact the return value is just a state save. If we decide to reset the counter in an external function, then we will spread the responsibility for the variable (and yield + reset) between two functions.

But we could extract wait + reset code into a separate method.